CMakeLists.txt

```diff
@@ -59,6 +60,11 @@ execute_process(
 )
 string(STRIP ${PYTHON_SYS_PATH} PYTHON_SYS_PATH)
 
+if (USE_LRU_MASTER)
+  add_compile_definitions(USE_LRU_MASTER)
+  add_compile_definitions(LRU_MAX_CAPACITY=1000)
```

Comment:
we'd better move it to command arguments, so that we can change it on demand.

Reply:
now fixed